useKeyPress/test.jsx

```diff
@@ -0,0 +1,49 @@
+//And here is a test case to validate the hook:
+
+import React from "react";
+import { render, fireEvent, cleanup } from "@testing-library/react";
+import useKeyPress from "./useKeyPress";
+
+describe("useKeyPress", () => {
+  afterEach(cleanup);
```

Comment:
I'm glad you're thing about cleanup.  That's very mature.  However, I'm pretty sure that this automatically gets taken care of, so I don't think it's necessary.

Reply:
https://testing-library.com/docs/react-testing-library/api#cleanup